utils/chaos-recommender/requirements.txt

```diff
@@ -0,0 +1 @@
+pandas
```

Comment:
Hi @mudverma, we might want to restrict the pandas version to < 2.0.0 to support the current functionality and also add prometheus_api_client to the list. Thoughts?

Reply:
sorry for the oversight @chaitanyaenr . Adding this.